### DISCUSSION TITLE
Please, add this next js docs theme

### DISCUSSION THREAD

**Discussion by GrigoriiPrudnikov** [zed-industries/zed#18566] [Feature Requests]
[https://github.com/wilmarjongkind/nextjs-docs-theme/tree/main](https://github.com/wilmarjongkind/nextjs-docs-theme/tree/main)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
